python/lsst/afw/typehandling/_GenericMap.cc

```diff
@@ -126,7 +126,8 @@ void declareGenericMap(utils::python::WrapperCollection& wrappers, std::string c
         // __iter__ easier to implement in Python
         cls.def("__len__", &Class::size);
         cls.def("__bool__", [](Class const& self) { return !self.empty(); });
-        cls.def("_keys", &Class::keys);  // Private to let a proper view be defined in Python
+        // Private to let a proper view be defined in Python
+        cls.def("_keys", &Class::keys, py::return_value_policy::reference_internal);
```

Comment:
It [already existed](https://github.com/lsst/afw/blob/master/python/lsst/afw/typehandling/_GenericMap.py#L56). I *think* I left it in because there were some object-lifetime issues from wrapping `keys` directly, but I should confirm that.

Now that I think of it, as of bd252207c10ae625650378945715272cc2b00e19 I don't use `_keys` at all, so I should get rid of it either way.

Reply:
I remembered wrong; the real problem was that pybind11 always copies `std::vector` into a Python `list`, so `keys()` doesn't act like a view if you wrap it directly.

Reply:
Ah. Could you please document that fact, to remind others?

Reply:
Is the [current comment](https://github.com/lsst/afw/pull/466/files#diff-a248b20ec9ffaa9f7c8675d0fce06a88R131) too vague?

Reply:
That's helpful, thanks.